cmd/buildutil/info.go

```diff
@@ -386,7 +386,7 @@ func ParseS3URL(raw string) (*S3URL, error) {
 
 	return &S3URL{
 		Bucket: u.Host,
-		Key:    path.Clean("/" + u.Path),
+		Key:    strings.TrimPrefix(path.Clean(u.Path), "/"),
```

Comment:
Why?

Reply:
That's what my Slack rant was about. The `Key` must not start with a slash as this is interpreted as a separate directory.